core/lib/types/Cargo.toml

```diff
@@ -21,6 +21,8 @@ zksync_config = { path = "../config" }
 codegen = { git = "https://github.com/matter-labs/solidity_plonk_verifier.git", branch = "dev" }
 zkevm_test_harness = { git = "https://github.com/matter-labs/era-zkevm_test_harness.git", branch = "v1.3.3" }
 zksync_protobuf = { version = "0.1.0", git = "https://github.com/matter-labs/era-consensus.git", rev = "5b3d383d7a65b0fbe2a771fecf4313f5083be9ae" }
+zkevm_test_harness_1_4_1 = { package = "zkevm_test_harness", git = "https://github.com/matter-labs/era-zkevm_test_harness.git", rev = "12f717f12520b2ab87458f125950d7b19e6eb74e" }
```

Comment:
we can most likely move this out of types, maybe utils is a better place for it?

Reply:
I think that the platform team would probably prefer to see such functions in a mulitvm-like crate. Or even better (if we consider that these functions won't change) separate the blob-related functions from the harness crate and import such separate crate here. 

In any case, I would suggest to firstly consult with Danil/Igor on this matter since they have a broader picture on this refactoring in their head